client/src/app/pages/migration-waves/migration-waves.tsx

```diff
@@ -400,14 +425,20 @@ export const MigrationWaves: React.FC = () => {
                             </DropdownItem>,
                             <DropdownItem
                               key="bulk-delete"
+                              isDisabled={selectedItems.length === 0}
                               onClick={() => {
+                                let deleteMigrationWaves: Promise<MigrationWave>[] =
+                                  [];
```

Comment:
```suggestion
                                const deleteMigrationWaves: Promise<MigrationWave>[] =
                                  [];
```
const is fine here since we do not plan to reassign references to the object.